packages/server/tests/helpers/assertions.ts

```diff
@@ -398,4 +399,25 @@ export default class Assertions {
     expect(address).to.match(/(\b0x[a-f0-9]{40}\b)/g, 'matches evm address format');
     expect(address).to.match(/(\b0x(0){15})/g, 'contains 15 consecutive zeros');
   };
+
+  static validateResultDebugValues = (result, excludedValues, nestedExcludedValues, expectedResult) => {
```

Comment:
@anastasiya-kovaliova Please add types as well - for the parameters, the return value and the variables in the function. I know we do not have them in a lot of places, but we should start adding them